src/generic/infobar.cpp

```diff
@@ -236,7 +236,7 @@ void wxInfoBarGeneric::ShowMessage(const wxString& msg, int flags)
     // notice the use of EscapeMnemonics() to ensure that "&" come through
     // correctly
     m_text->SetLabel(wxControl::EscapeMnemonics(msg));
-
+	m_text->Wrap( GetClientSize().GetWidth() );
```

Comment:
```suggestion
    m_text->Wrap( GetClientSize().GetWidth() );
```
(removing hard TAB...)